src/libs/EmojiUtils.js

```diff
@@ -245,6 +245,66 @@ function suggestEmojis(text, limit = 5) {
     return [];
 }
 
+/**
+ * Validates that this message contains emojis
+ *
+ * @param {String} message
+ * @returns {Boolean}
+ */
+function hasEmojis(message) {
+    const trimmedMessage = Str.replaceAll(message.replace(/ /g, ''), '\n', '');
+    return Boolean(trimmedMessage.match(CONST.REGEX.EMOJIS));
+}
+
+/**
+ * Validates that this message contains emojis
+ * @param {String} text
+ * @returns {Array}
+ */
+function containsEmoji(text) {
```

Comment:
This is no longer an accurate function name.

Reply:
I changed this to `getAllEmojiFromText` what do you think?